src/app/backend/resource/secret/secretdetail.go

```diff
@@ -36,9 +36,6 @@ type SecretDetail struct {
 
 	// Used to facilitate programmatic handling of secret data.
 	Type api.SecretType `json:"type"`
-
-	// Determines if the secret is hidden type in case of opaque and basicAuth types.
-	IsHiddenType bool `json:"isHiddenType"`
```

Comment:
Yes, there's change in behavior here, yes. I propose hiding all secrets by default, and only showing some of them we pick by hand (possibly none). This is to follow security "principle" similar to what you have in, e.g., firewalls, i.e., you disallow all traffic and allow only selected ports and protocols.

Does this make sense?